scout/constants/variant_tags.py

```diff
@@ -295,6 +295,24 @@
 }
 
 MANUAL_RANK_OPTIONS = {
```

Comment:
I was just comparing this list with the [clinsigs recommended](https://www.ncbi.nlm.nih.gov/clinvar/docs/clinsig/) by ClinVar, just for fun.

Why not including all criteria from ClinGen instead, it's more like a concept, instead of 3 useful but random terms from it.. Some of these terms make more sense to me than others from our list :)

<img width="897" alt="image" src="https://user-images.githubusercontent.com/28093618/212836462-af4f20e0-a9be-407e-951b-4f5b8f88c02e.png">

Reply:
Good idea re-checking. This "system" we have is for historical and practical reasons modelled after ESHG and DeCipher in particular, by way of the aCGH classifications we use for SVs. The risk factors are not included from there though, rather they are picked a bit ad hoc from ClinVar and the ACMG/AMP variant classification system body text (they are not in their recommendation as such yet). There is definitely merit to naming them "risk alleles" as ClinGen do; that would be my formal go-to. As the issue also asked for "risk factor", which is also what most people say, I stuck with it, and used the ClinGen language in the description to clarify.

The additional "low penetrance" ones could potentially be useful: they are not called for, but the ClinGen make a good recommendation for them in https://clinicalgenome.org/site/assets/files/4531/clingenrisk_terminology_recomendations-final-02_18_20.pdf. Let's include them!